website/docs/operators/update_operators/field-update-operators.md

```diff
@@ -0,0 +1,334 @@
+| [`$unset`](#unset)             | Deletes the records of a field from a document                                              |
+| [`$inc`](#inc)                 | Increments a given field's value                                                            |
+| [`$mul`](#mul)                 | Multiplies a given field’s value by a specific value                                        |
+| [`$rename`](#rename)           | Renames a given field with another name                                                     |
+| [`$min`](#min)                 | Updates a particular field only when the specified value is lesser than the specified value |
+| [`$max`](#max)                 | Updates a particular field only when the specified value is higher than the specified value |
+| [`$currentDate`](#currentdate) | Specifies the current date and time as the value of a given field                           |
+| `$setOnInsert`                 | Inserts elements into an array only if they don't already exist                             |
+
+For the examples in this section, insert the following documents into the `employees` collection:
+
+```js
+db.employee.insertOne({
+    name: "John Doe",
+    age: 35,
+    email: "johndoe@example.com",
+    phone: "123-456-7890",
+    address: {
+        street: "123 Main St",
+        city: "Anytown",
+        state: "CA",
+    },
+    salary: 50000,
+    jobTitle: "Manager",
+    startDate: new Date("2021-01-01"),
+    endDate: null
+})
+```
+
+## $set
+
+The `$set` operator updates the value of a specified field and if the field does not exist, the `$set` operator creates a new field and adds it to the document.
+
+The below query is an example that updates the value of the `city` field in the `address` embedded document.
+
+```js
+db.employee.updateOne(
+    { name: "John Doe" },
+    { $set: {
+        "address.city": "New York",
+        "address.zip": "12345",
+        } }
+)
+```
+
+The above query updates the value of the `city` field in the `address` embedded document and adds a new field `zip` to it.
+
+This is the updated document:
+
+```js
+[
+  {
+    _id: ObjectId("640a603558955e0e2b57c00d"),
+    name: 'John Doe',
+    age: 35,
+    email: 'johndoe@example.com',
+    phone: '123-456-7890',
+    address: {
+      street: '123 Main St',
+      city: 'New York',
+      state: 'CA',
+      zip: '12345',
+    },
+    salary: 50000,
+    jobTitle: 'Manager',
+    startDate: ISODate("2021-01-01T00:00:00.000Z"),
+    endDate: null
+  }
+]
+```
+
+## $unset
+
+The `$unset` operator deletes the specified field from a document and if the field is not present, the `$unset` operator will not do anything.
+
+The below query deletes the `zip` field from the embedded document `address`.
+
+```js
+db.employee.updateOne(
+    { name: "John Doe" },
+    { $unset: { "address.zip": "" } }
+)
+```
+
+Below is the updated document showing the updated document without the pox :
```

Comment:
Sorry, what is `pox`?

Reply:
Thanks for pointing this out. Initially named it pox (po box, instead of zip). Updating it now.